net-misc/gns3-gui/gns3-gui-2.0.3.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_4 )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Graphical Network Simulator"
+HOMEPAGE="http://www.gns3.net/"
+SRC_URI="https://github.com/GNS3/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="test"
+
+#net-misc/gns3-server version should always match gns3-gui version
+
+RDEPEND="
+	>=dev-python/libcloud-0.15.1[${PYTHON_USEDEP}]
+	>=dev-python/ws4py-0.3.4[${PYTHON_USEDEP}]
+	>=dev-python/requests-2.6.0[${PYTHON_USEDEP}]
+	>=dev-python/paramiko-1.15.1[${PYTHON_USEDEP}]
+	=net-misc/gns3-server-$PVR[${PYTHON_USEDEP}]
+	dev-qt/qtgui:5
+	dev-qt/qtsvg:5
+	dev-python/PyQt5[gui,network,svg,widgets,${PYTHON_USEDEP}]
```

Comment:
I really fail to see a point in depending on Qt directly when it's using pyqt. Does it build C++ code using Qt?